services/src/atdd-staging/src/main/java/org/openkilda/atdd/staging/steps/NorthboundSteps.java

```diff
@@ -107,4 +107,10 @@ public void removeMeter(String meterAlias, String switchAlias) {
     public void removeMeterResponseIsSuccessful() {
         assertTrue(deleteMeterResponse.isDeleted());
     }
+
+    @And("^remove meter -1 from '(.*)'$")
```

Comment:
For what purpose you added that?

Reply:
Oops.. It was old test implementation. I forgot to remove it